op-node/rollup/driver/sequencer.go

```diff
@@ -91,6 +91,7 @@ func (d *Sequencer) CompleteBuildingBlock(ctx context.Context) (*eth.ExecutionPa
 	if err != nil {
 		return nil, fmt.Errorf("failed to complete building on top of L2 chain %s, error (%d): %w", d.buildingOnto.HeadBlockHash, errTyp, err)
 	}
+	d.buildingID = eth.PayloadID{}
```

Comment:
Should this be part of its own PR?

Reply:
The difference is clear with the tests in this PR, but behavior is equivalent outside of action testing, so I think we just keep it here.